go-bindings/llmq_example_test.go

```diff
@@ -0,0 +1,277 @@
+package blschia_test
+
+import (
+	"bytes"
+	bls "github.com/dashpay/bls-signatures/go-bindings"
```

Comment:
Not technically wrong, but looks kinda weird having non-std lib imports mixed w/std lib ones.

Reply:
seems like the go linter took care of this haha